src/js/components/VariableEditor.js

```diff
@@ -32,307 +37,389 @@ class VariableEditor extends React.Component {
-            {
-                type == 'array'
-                ? (
-                <span {...Theme(theme, 'array-key')}
-                key={variable.name + '_' + namespace}>
-                    {variable.name}
-                    <div {...Theme(theme, 'colon')}>:</div>
-                </span>
-                )
-                : (
-                <span>
-                    <span {...Theme(theme, 'object-name')}
-                    class="object-key"
-                    key={variable.name + '_' + namespace}>
-                        <span style={{verticalAlign:'top'}}>"</span>
-                        <span style={{display:'inline-block'}} >{variable.name}</span>
-                        <span style={{verticalAlign:'top'}}>"</span>
+            <div
+                {...Theme(theme, "objectKeyVal", {
+                    paddingLeft: indentWidth * singleIndent
+                })}
+                class="variable-row"
+                key={variable.name}
+            >
+                {type == "array" ? (
+                    <span
+                        {...Theme(theme, "array-key")}
+                        key={variable.name + "_" + namespace}
+                    >
+                        {variable.name}
+                        <div {...Theme(theme, "colon")}>:</div>
                     </span>
-                    <span {...Theme(theme, 'colon')}>:</span>
-                </span>
-                )
-
-            }
-            <div class="variable-value"
-
-            onClick={onSelect === false && onEdit === false ? undefined : (e)=>{
-                let location = [...namespace]
-                if (e.ctrlKey) {
-                    this.prepopInput(variable);
-                } else if (editMode === false) {
-                    location.shift()
-                    onSelect({...variable, namespace: location})
-                }
-            }}
-            {...Theme(theme, 'variableValue', {cursor: onSelect === false ? 'default' : 'pointer'})}>
-                {this.getValue(variable, this.props, editMode)}
+                ) : (
+                    <span>
+                        <span
+                            {...Theme(theme, "object-name")}
+                            class="object-key"
+                            key={variable.name + "_" + namespace}
+                        >
+                            <span style={{ verticalAlign: "top" }}>"</span>
+                            <span style={{ display: "inline-block" }}>
+                                {variable.name}
+                            </span>
+                            <span style={{ verticalAlign: "top" }}>"</span>
+                        </span>
+                        <span {...Theme(theme, "colon")}>:</span>
+                    </span>
+                )}
+                <div
+                    class="variable-value"
+                    onClick={
+                        onSelect === false && onEdit === false
```

Comment:
this is the only functional change to this file.  

the rest was updated because I started using [prettier](https://github.com/prettier/prettier) sublime plugin which auto-formats on save